### PR TITLE
Just enable puppet4 repo no need to instal rpms

### PR DESCRIPTION
**Pull request by lpramuk**
- renamed install_puppet4() to enable_puppet4()
- removed fixed BZ #1381081 workaround
- do not install p4 puppet-agent just enable repo

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1255,17 +1255,14 @@ def install_prerequisites():
     manage_daemon('start', 'ntpd', warn_only=True)
 
 
-def install_puppet4(puppet4_repo=None):
+def enable_puppet4(puppet4_repo=None):
     """Task to install puppet4 prior Satellite installation"""
```
Comment (kbidarkar) [inline automation_tools/__init__.py]: I think, we need to update it to reflect `enable`, instead of `install`.

**Comment by kbidarkar** [comment]
ACK

**Comment by lpramuk** [comment]
I removed whole ```install_puppet4()``` task as it turned into one simple ```create_custom_repo()``` command

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -1255,19 +1255,6 @@ def install_prerequisites():
     manage_daemon('start', 'ntpd', warn_only=True)
 
 
-def install_puppet4(puppet4_repo=None):
-    """Task to install puppet4 prior Satellite installation"""
-    # Remove any puppet3 server to avoid conflicts
-    run('yum -y remove puppet-server', warn_only=True)
-    create_custom_repos(puppet4_repo=puppet4_repo)
```
Comment (lpramuk) [inline automation_tools/__init__.py]: this one remained so I removed whole task and put this one in place of install_puppet4()

File: automation_tools/__init__.py [hunk 3]
```diff
@@ -1963,7 +1950,10 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     if satellite_version in ('6.3', 'downstream-nightly'):
         puppet4_repo = os.environ.get('PUPPET4_REPO')
         if puppet4_repo:
-            execute(install_puppet4, puppet4_repo=puppet4_repo, host=host)
+            # Just enable puppet4 repo, no need to install puppet4 since
+            # puppet-agent is pulled in by rpm deps
+            # puppetserver is pulled in by installer
+            execute(create_custom_repos, puppet4_repo=puppet4_repo, host=host)
```
Comment (lpramuk) [inline automation_tools/__init__.py]: here

**Comment by kbidarkar** [comment]
ACK

**Pull request merged**